core/queriec.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+#include <stddef.h>
+#include <stdarg.h>
+
+#include "queriec.h"
+
+void
+queriec_init(struct queriec_obj *obj, size_t size, size_t offset)
+{
+    obj->size = size;
+    obj->status = 0;
+    obj->offset = offset;
+}
+
+void
+_queriec_add(struct queriec_obj *obj, char *query, char *letter, char key[], size_t keySize, char value[], size_t valueSize)
```

Comment:
```suggestion
_queriec_add(struct queriec_obj *obj, char *query, char *letter, char key[], size_t key_sz, char value[], size_t value_sz)
```
It's better to pick either `camelCase` or `snake_case` and stick with it, not doing a mixing of both!